darts/models/forecasting/tft_model.py

```diff
@@ -706,6 +732,9 @@ def __init__(
         likelihood
             The likelihood model to be used for probabilistic forecasts. By default, the TFT uses
             a ``QuantileRegression`` likelihood.
+        norm_type: str
```

Comment:
Could we optionally allow providing a custom `nn.Module` ? That might be nice to quickly test potential new variants of normalisation layers in the future. WDYT?

Reply:
Done